tree/tree/test/ImplicitMT.cxx

```diff
@@ -28,4 +29,32 @@ TEST(TTreeImplicitMT, flushBaskets)
    gSystem->Unlink(ofileName);
 }
 
+//# 8720
+TEST(TChainImplicitMT, propagateToTTree)
+{
+   ROOT::DisableImplicitMT();
+
+   const auto fname0 = "propagateToTTree0.root";
+   const auto fname1 = "propagateToTTree0.root";
```

Comment:
The two filename seem to be the same.  Is that the intent?

Reply:
no. Fixing.